src/components/InstanceDetailsList.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from '@patternfly/react-core';
+
+import { getDateTime } from '../utils/date';
+import { cloudProviderValueToLabel } from '../utils/cloudProvider';
+import { regionValueToLabel } from '../utils/region';
+
+function InstanceDetailsList({ instance }) {
+  return (
+    <DescriptionList isHorizontal>
+      <DescriptionListGroup>
+        <DescriptionListTerm>ID</DescriptionListTerm>
+        <DescriptionListDescription>{instance.id}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Owner</DescriptionListTerm>
+        <DescriptionListDescription>
+          {instance.owner}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Time created</DescriptionListTerm>
+        <DescriptionListDescription>
+          {getDateTime(instance.created_at)}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Time updated</DescriptionListTerm>
+        <DescriptionListDescription>
+          {getDateTime(instance.updated_at)}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Cloud provider</DescriptionListTerm>
+        <DescriptionListDescription>
+          {cloudProviderValueToLabel(instance.cloud_provider)}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Region</DescriptionListTerm>
+        <DescriptionListDescription>
+          {regionValueToLabel(instance.region)}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>Sensor Host URL</DescriptionListTerm>
+        <DescriptionListDescription>-</DescriptionListDescription>
```

Comment:
Is the sensor host URL not defined on the instance object? (TBH I didn't even know sensor was accessible via URL.)

Reply:
So based on the wording by the backend team we have two URLs:

1. The Central UI URL
2. The Central URL for Sensor

Hopefully, that makes more sense. Seems like they've settled on the following field names:

1. `centralUIURL`
2. `centralDataURL`

I'll use these names in the UI for now and once their changes come in it'll show the correct data

Reply:
I changed the order to be the same as the mock as well